ajax/libs/rrssb/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "rrssb",
+  "filename": "rrssb.min.js",
+  "version": "1.14.0",
+  "author": "Daniel Box <daniel@kurtnoble.com> (http://github.com/kni-labs)",
```

Comment:
Let's use `https`. This can let people browser the website more security.
`http://github.com/kni-labs` --> `https://github.com/kni-labs`
Thanks